frontend/pages/interest/thank-you.tsx

```diff
@@ -0,0 +1,26 @@
+import { NextPage } from 'next';
+import Lottie from 'lottie-react';
+import orangeTick from '../../assets/orangeTickLottie.json';
+import { Box, Container, Stack, Typography } from '@mui/material';
+import { stackSx } from '../../styles/interest/thankyou';
+
+const ThankYou: NextPage = () => {
+  return (
+    <div>
+      <Container component="main" maxWidth="md">
```

Comment:
Can we make the contents centered? Also, the tick is way too big.

![image](https://user-images.githubusercontent.com/53614454/195811005-04ac5d1a-3d70-4ac2-ad17-4d8085c13df2.png)